pages/specification/migration/_index.md

```diff
@@ -3,5 +3,78 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Migrations are like version control for your system, allowing your team to define and share the schema progressions. If you have ever had to figure out when a keyword was introduced and how it behaved, you've faced the problem that migration guides solve.
+
+If you're moving from one Draft to another, learn how to use our migration guides and tooling ([alterschema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you going.
+
+| All Keywords          | Specification          | Draft introduction | Removed | Changed                       |
+| --------------------- | ---------------------- | ------------------ | ------- | ----------------------------- |
+| $anchor               | Core                   | 2019-09            | No      | No (updated in 2020)          |
+| $comment              | Core                   | 07                 | Yes     | -                             |
+| $id                   | Core                   | 06                 | No      | Updated from id               |
+| $defs                 | Core                   | 2019-09            | No      | Updated from definitions      |
```

Comment:
All of these keywords that were "updated from" other keywords should probably be marked as removed? i.e. `definitions` is supposed to be gone after the introduction of `$defs`

Reply:
Maybe something like "replaces" and "replaced by" instead of "updated to/from".